### PR TITLE
Testcase

### PR DESCRIPTION
**Pull request by kevindiu**

### CODE REVIEW

File: config/config.go [hunk 1]
```diff
@@ -139,13 +138,12 @@ type Authorization struct {
 func New(path string) (*Config, error) {
 	f, err := os.OpenFile(path, os.O_CREATE|os.O_RDONLY, 0600)
 	if err != nil {
-		return nil, errors.Wrap(err, "OpenFile failed")
+		return nil, errors.Wrap(err, "config read failed")
```
Comment (kpango) [inline config/config.go]: I think os.OpenFile is not read function, just open file desctiptor only,
so, `os.OpenFile` or `open file` is better than `config read`

Comment (kevindiu) [reply config/config.go]: I am thinking line 149,
```go
f, err := os.OpenFile(path, os.O_CREATE|os.O_RDONLY, 0600)
```
maybe it is better to remove os.O_CREATE flag while open the file??

Comment (kpango) [reply config/config.go]: Opps,

I think so, can you remove it?

File: usecase/providerd_test.go [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+		args    args
+		want    AuthorizationDaemon
+		wantErr bool
+	}{
+		//		{
+		//			name: "new success",
+		//			args: args{
+		//				cfg: config.Config{},
+		//			},
+		//		},
```
Comment (kpango) [inline usecase/providerd_test.go]: what's this?

File: config/config.go (hunk 1 above)
Comment (kevindiu) [reply config/config.go]: Sure :)

**Comment by kevindiu** [comment]
f

**Pull request closed without merging**